client/state/connectionMiddleware/middleware.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+const connections = {};
```

Comment:
Yes, helps a lot! (though I read more about it and understood it like a minute after I wrote the dull comment..). Maybe you could post this to some guide? (e.g. the [Calypso JS guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md)). Valuable info.